### DISCUSSION TITLE
Tile-style card for areas

### DISCUSSION THREAD

**Discussion by bieniu** [home-assistant/frontend#20159] [Dashboard] [answered]
Currently, it is not possible to create cards for areas that would resemble Tile cards in style.

I propose creating a special Tile card that would be adapted to areas.
Such a card should:
- be able to select sensor states from a given area as a card `content state`
- be able to use the motion/smoke/flood/window binary sensor from a given area as a badge

Example:

![obraz](https://github.com/home-assistant/frontend/assets/478555/b1420bbe-0f39-4c4b-8504-f1a94598562d)

**Comment by BBE-FR** [comment]
feature request similar to mine: https://github.com/home-assistant/frontend/discussions/19930 I fully support!

**Comment by bieniu** [comment, marked as answer]
Closed via https://github.com/home-assistant/frontend/pull/25802
